ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileChecksPage.tsx

```diff
@@ -38,8 +98,49 @@ function ProfileChecksPage() {
                 <CoveragesToggleGroup tableView="checks" />
             </PageSection>
             <PageSection variant="default">
-                <PageSection variant="light" component="div">
-                    <Title headingLevel="h2">Profile results</Title>
+                <PageSection variant="light" className="pf-v5-u-p-0">
+                    <Toolbar>
+                        <ToolbarContent>
+                            <ToolbarGroup className="pf-v5-u-w-100">
+                                <Title headingLevel="h2" className="pf-v5-u-py-md">
+                                    Profile results
+                                </Title>
+                            </ToolbarGroup>
+                            <ToolbarGroup className="pf-v5-u-w-100">
+                                <ToolbarItem className="pf-v5-u-flex-1">
+                                    <CompoundSearchFilter
+                                        config={searchFilterConfig}
+                                        searchFilter={searchFilter}
+                                        onSearch={onSearch}
+                                    />
+                                </ToolbarItem>
+                                <ToolbarItem align={{ default: 'alignRight' }}>
+                                    <CheckStatusDropdown
+                                        searchFilter={searchFilter}
+                                        onSelect={onSelectCheckStatus}
+                                    />
+                                </ToolbarItem>
+                            </ToolbarGroup>
+                            <ToolbarGroup className="pf-v5-u-w-100">
+                                <SearchFilterChips
+                                    filterChipGroupDescriptors={[
+                                        {
+                                            displayName: 'Profile Check',
+                                            searchFilterName: 'Compliance Check Name',
+                                        },
+                                        {
+                                            displayName: 'Cluster',
+                                            searchFilterName: 'Cluster',
+                                        },
+                                        {
+                                            displayName: 'Compliance Status',
+                                            searchFilterName: 'Compliance Check Status',
+                                        },
+                                    ]}
```

Comment:
👍 I've got a similar concept in a draft PR for VM 2.0 we could use if it would work here as well https://github.com/stackrox/stackrox/pull/11333/files#diff-8f9ae03492ab834298ab13ae1020923209acce70a99b1903110f52091afee9b9R107